Sources/PureSwiftUI/ViewModifiers/Font/DynamicallyScaledFontViewModifier.swift

```diff
@@ -14,7 +14,11 @@ public struct DynamicallyScalingFontViewModifier: ViewModifier {
     var weight: Font.Weight?
 
     public func body(content: Content) -> some View {
+        #if os(macOS)
+        let scaledSize = CGFloat(17)
+        #else
         let scaledSize = UIFontMetrics.default.scaledValue(for: size)
+        #endif
         return content.font(createFont(scaledSize: scaledSize))
     }
```

Comment:
I wasn’t sure what to replace this with either on macOS, I wonder what the macCatalyst implementation calls through to in AppKit. Would be great if you could take a look, perhaps having a hardcoded value is okay.

Reply:
Dynamic type does not exist on macOS. The docs for the `sizeCategory` environment value (now renamed `dynamicTypeSize`) states that:

> On macOS, this value cannot be changed by users and does not affect the text size

Just make this whole modifier unavailable on macOS.

Reply:
yes completely agree. Once this PR is applied, I'll move the conditional compilation to include the entire extension.